package.json

```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "@types/node": "^20.12.7",
     "@types/yargs": "^15.0.10",
-    "aws-cdk": "^2.182.0",
+    "aws-cdk": "2.179.0",
```

Comment:
huh?

Reply:
See the notes on the PR. It's the thing we ran into when we were deploying on Sunday. yarn doesn't have a record of the version we used to use. 🤷